lib/rules/lint-no-whitespace-for-layout.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'Excess whitespace detected.';
+
+module.exports = class NoWhitespaceForLayout extends Rule {
+  visitor() {
+    return {
+      TextNode(node) {
```

Comment:
should this rule be ignored inside `<pre>`, `<script>`, `<style>` tags?

Reply:
Should it warn for common cases like
`{{this.name}} years`

`age is {{this.years}`